lib/alice/adapters/noop.ex

```diff
@@ -13,35 +15,31 @@ defmodule Alice.Adapters.NoOp do
   """
   use Alice.Adapter
 
-  @doc false
-  def init({bot, _opts}) do
-    send(self(), :connected)
+  def connect(bot, _opts) do
     {:ok, bot}
   end
 
-  @doc false
-  def handle_cast({:reply, msg}, bot) do
-    IO.puts "got reply message #{inspect msg} from bot #{inspect bot}"
-    {:noreply, bot}
+  def handle_connect(bot) do
```

Comment:
this should be `handle_connected`

Reply:
I changed my mind... this should be `handle_connect`
It makes more sense in the context of everything else.